### PR TITLE
[DEV-12292] Only take textColor into account if withoutBackgroundImage is true for standard widgets

### PR DESCRIPTION
**Pull request by katrinundfritz**
* Text color is always white when background image is displayed

### CODE REVIEW

File: src/components/preconfigured/Standard.tsx [hunk 1]
```diff
@@ -8,7 +8,10 @@ export const Standard = (props: WidgetProps) => {
   const params = new URLSearchParams(props.location.search)
   const demo = params.has('demo')
   const backgroundImage = params.get('withoutBackgroundImage') === 'true' ? null : defaultBackgroundImage
-  const textColor = params.get('textColor') || 'fff'
+  let textColor = 'ffffff'
+  if (params.get('withoutBackgroundImage') === 'true') {
+    textColor = params.get('textColor') || 'ffffff'
+  }
```
Comment (smnscp) [inline src/components/preconfigured/Standard.tsx:14]: For consistency with the ternary evaluation above I would do this as:
```suggestion
  const textColor = backgroundImage && params.get('textColor') ? params.get('textColor') : 'ffffff'
```

**Review by smnscp — CHANGES_REQUESTED** [review]

**Review by smnscp — APPROVED** [review]

File: src/components/preconfigured/Standard.tsx (hunk 1 above)
Comment (katrinundfritz) [reply src/components/preconfigured/Standard.tsx:14]: I leave it as is. Decision was approved by Simon.

Comment (smnscp) [reply src/components/preconfigured/Standard.tsx:14]: It’s also fine to leave it as is. (As we just found out that TS would require the expression to be a bit more complex, decreasing its readability.)

Comment (smnscp) [reply src/components/preconfigured/Standard.tsx:14]:
```suggestion
  const textColor = backgroundImage && params.get('textColor') || 'ffffff'
```
This should have worked 🤦

**Pull request merged**